lib/Conversion/LLHDToLLVM/LLHDToLLVM.cpp

```diff
@@ -1907,21 +1906,10 @@ void LLHDToLLVMLoweringPass::runOnOperation() {
   populateCombToArithConversionPatterns(converter, patterns);
   arith::populateArithToLLVMConversionPatterns(converter, patterns);
 
-  target.addLegalDialect<LLVM::LLVMDialect>();
   target.addLegalOp<ModuleOp>();
 
-  // Apply a full conversion to remove unrealized conversion casts.
   if (failed(applyFullConversion(getOperation(), target, std::move(patterns))))
-    signalPassFailure();
-
-  patterns.clear();
-
-  mlir::populateReconcileUnrealizedCastsPatterns(patterns);
-  target.addIllegalOp<UnrealizedConversionCastOp>();
-
-  // Apply the full conversion.
-  if (failed(applyFullConversion(getOperation(), target, std::move(patterns))))
-    signalPassFailure();
+    return signalPassFailure();
 }
```

Comment:
`populateReconcileUnrealizedCastsPatterns` was removed and we have to instead rely on running the `reconcile-unrealized-casts` pass separately.